test/parallel/test-dotenv-node-options.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+require('../common');
+const assert = require('node:assert');
+const { spawnSync } = require('node:child_process');
+
+const envFilePath = '../fixtures/dotenv/node-options.env';
+
+{
+  // Make sure we have access to permission scope.
+  const code = '"require(\'assert\').assert(process.permission.has(\'fs.read\'))\'"';
+  const child = spawnSync(process.execPath,
```

Comment:
If you refactor this file to use the `spawnPromisified` helper, these child processes can run in parallel instead of in sequence, making this test faster.